### PR TITLE
feat: add storybook

### LINKED ISSUE #2: Add storybook
Let's use https://storybook.js.org/ for a better development experience

### PR DESCRIPTION
**Pull request by lfjnascimento**
## Description

- Install storybook;
- Do not add the default demo components other than button;
- Do not add the @storybook/addon-onboarding add-on as it is only for the initial guided tour

## Related Issues
- Closes #2

### CODE REVIEW

File: dashboard/.gitignore [hunk 1]
```diff
@@ -22,3 +22,5 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+*storybook.log
```
Comment (mari1912) [inline dashboard/.gitignore:26]: nit: add a new line

**Review by mari1912 — COMMENTED** [review]
I think `components` is a better name than `stories` to the folder the button is in. Other than that, it looks good

File: dashboard/.gitignore (hunk 1 above)
Comment (lfjnascimento) [reply dashboard/.gitignore]: I created `src/component` and placed the Button component and style there. As soon as we add our components we can delete this button example

**Comment by lfjnascimento** [comment]
> I think `components` is a better name than `stories` to the folder the button is in. Other than that, it looks good

I created src/component and placed the Button component and style there. As soon as we add our components we can delete this button example

**Review by mari1912 — APPROVED** [review]

**Pull request merged**